modules/ztp-high-security-issues-addressed.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-security-hardening.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-addressed-security-issues_{context}"]
+= Security hardening measures implemented
+
+== Disabling USB boot, wireless and bluetooth access  
+
+To prevent unauthorized modifications to the operating system and to reduce the risk of physical attack vectors, USB boot, wireless, and Bluetooth are disabled in the BIOS settings. These setting are not controlled through ZTP configuration. For example, disabling USB boot prevents attackers from booting from a USB drive and gaining unauthorized access to the system. The recommended firmware configuration is described in "Configuring host firmware for low latency and high performance".
+
+== Preventing unauthorized Reboots
+ 
+Disabling `CtrlAltDelBurstAction` and `Ctrl+Alt+Del` prevents unauthorized reboots.
+
+== Restricting direct SSH access
+  
+Direct SSH access to {product-title} is limited by disabling `root` login.
+
+== Adhering to the principle of least privilege with RBAC
+  
+Enforce strict authentication policies by disabling logins for accounts without passwords, ensuring compliance with security best practices and reducing unauthorized access risks.
+
+== Ensure that the RBAC setup follows the principle of least privilege
```

Comment:
I am kind of confused because to me "Adhering to the principle of least privilege with RBAC" and "Ensure that the RBAC setup follows the principle of least privilege" seem to be the same thing? I would combine these 2 sections. However, maybe you have to have separate headings since these are standards and you need to have a heading for each one? If so, then disregard my suggestion.